src/docs/apex-import.md

```diff
@@ -1,10 +1,11 @@
 # apex-import
 
-Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios.
+Using Apex in LWC Offline-enabled mobile apps requires additional considerations to ensure proper functioning in offline scenarios. Consider using GraphQL for mobile offline use cases.
```

Comment:
I think this is good for now, though in the future it would be nice to also delineate the "additional considerations" for Apex in more detail.